README.md

```diff
@@ -15,18 +15,23 @@ Command-line interface for recheck.
       --help      Display this help message.
       --version   Display version info.
 Commands:
-  help     Display help information about the specified command.
-  version  Display version info.
-  diff     Display given differences.
-  commit   Accept given differences.
-  ignore   Ignore given differences.
+  help         Display help information about the specified command.
+  version      Display version info.
+  diff         Display given differences.
+  commit       Accept given differences.
+  ignore       Ignore given differences.
+  completion   Generate and display auto completion script.
 ```
 
 
 ## Download and Installation
 
 The recheck CLI is still under development, but we will soon release a minimal working solution.
 
+## Command Line Completion
```

Comment:
## Auto-Completion for Bash and ZSH

You can obtain an auto-completion script for Bash and ZSH via the `completion` command. Simply add the resulting output to your `.bash_profile` or `.bashrc`, for example:

```bash
$ echo "source <(recheck completion)" >> ~/.bashrc
```